gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Made changes to Handle Not Found Error and Session timeout set to 12 hrs

How to test:
-Negative testing
Clone the repo from develop and do the following
python manage.py makemigrations
python manage.py migrate
python manage.py runserver
Login as a user and click on events link. It redirects you to events.html currently it will have no events
Go to the command prompt you should see an error saying Not Found:/ None

-Positive testing
Clone the repo from error-not-found branch and do the following
python manage.py makemigrations
python manage.py migrate
python manage.py runserver
Login as a user and click on events link. It redirects you to events.html currently it will have no events
Go to the command prompt you will no longer see the error
